native/packages/react-native-bpk-component-button/src/styles.js

```diff
@@ -127,9 +127,14 @@ const modifiers = {
     },
   }),
   iconOnly: StyleSheet.create({
-    container: {
-      width: spacingXl,
-    },
+    container: Platform.select({
+      ios: () => ({
+        width: spacingXl,
+      }),
+      android: () => ({
+        width: buttonHeight,
+      }),
+    })(),
```

Comment:
The button wasn't wide enough on Android, and therefore wasn't a circle. This makes sure it matches the button height to be round.

Reply:
Begs the question, why not `buttonHeight` on both platforms?

Reply:
Tbh I left it as is, as in my head this is the button component:

![](https://media.giphy.com/media/uKpWZU3VXLprW/giphy.gif)

But I'll give it a go now and see!

Reply:
It works, take your upvote and get out of here 🤣 (I'll push an update now!)